packages/zevm-contracts/scripts/zeta-swap/stress-swap.ts

```diff
@@ -0,0 +1,86 @@
+/* ********************************* */
+/* How to run this script 
+/* To run this script you have to setup the enviroment (yarn and setup .env) and run this command 
+/* ZETA_NETWORK=athens npx hardhat run scripts/zeta-swap/stress-swap.ts --network "polygon-mumbai" 
+/* In the example we use "polygon-mumbai", that's the source network. Can be any valid source network 
+/* Be sure to have enough gas tokens to pay all the tx 
+/* This script is to stress the node but it's not optimize for real use. May fail to enquee some tx sometimes but will enquee enough to test 
+/* ********************************* */
+
+import { BigNumber } from "@ethersproject/bignumber";
+import { formatEther, parseEther } from "@ethersproject/units";
+import { SignerWithAddress } from "@nomiclabs/hardhat-ethers/signers";
+import { getAddress } from "@zetachain/addresses";
+import { ethers } from "hardhat";
+import { network } from "hardhat";
+
+import { ChainToZRC20, isSwappableNetwork, SwappableNetwork, ZRC20Addresses } from "../systemConstants";
+import { getSwapData } from "./helpers";
+
+const swapToChain = async (
+  zetaSwapAddress: string,
+  tssAddress: string,
+  signer: SignerWithAddress,
+  destinationNetwork: SwappableNetwork,
+  value: BigNumber,
+  nonce: number
+) => {
```

Comment:
Can we change this for an object? Otherwise we may confuse arguments.

Reply:
done